javascript/ql/src/Security/CWE-089/examples/SqlInjectionFix.js

```diff
@@ -0,0 +1,12 @@
+const app = require("express")(),
+      pg = require("pg"),
+      pool = new pg.Pool(config);
+
+app.get("search", function handler(req, res) {
+  // GOOD: use parameters
+  var query2 =
+    "SELECT ITEM,PRICE FROM PRODUCT WHERE ITEM_CATEGORY=$1" + " ORDER BY PRICE";
```

Comment:
It's a bit strange to have the concatenation operator here, though I can see it makes it more obvious that `$1` is a literal replacement for the `'" + xxx + "'` part.

If you'd rather keep the operator, could we perhaps move the `ORDER BY PRICE` to the following line?